relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -1,3 +1,4 @@
+use crate::metrics_extraction::transactions::extract_http_status_code;
```

Comment:
The refactor of this method (moving it to `utils` module) is done in https://github.com/getsentry/relay/pull/2195, which should be merged before this PR. The reason I'm not stacking PRs is because it causes quite a lot of conflicts, that I don't consider worth resolving for a one-liner update.